readme.md

```diff
@@ -129,6 +129,7 @@ The [DICOM Standard](https://www.dicomstandard.org/) is _the_ international stan
 - [dicomViewerLib](https://github.com/fourctv/dicomViewerLib) - An Angular 9+ DICOMWeb viewer component, based on the [cornerstone.js](https://github.com/cornerstonejs/) project.
 - [DWV](https://github.com/ivmartel/dwv) - DICOM Web Viewer (DWV) is an open source zero footprint medical image viewer library. It uses only JavaScript and HTML5 technologies, meaning that it can be run on any platform that provides a modern browser (laptop, tablet, phone and even modern TVs).
 - [dwv-react](https://github.com/ivmartel/dwv-react) - Medical viewer using DWV and [React](https://react.dev/).
+- [NiiVue](https://github.com/niivue/niivue)  is a WebGL module that can be embedded into HTML, Vue, Angular, React, Electron and Capacitor frameworks. It provides scripting and drag and drop abilities to visualize and render voxel-based images (eg. DICOM, NIfTI), meshes, tractography streamlines and connectomes. [Live demos](https://niivue.github.io/niivue/) illustrate capabilities including [drawing](https://niivue.github.io/niivue/features/cactus.html) functions.
```

Comment:
```suggestion
- [NiiVue](https://github.com/niivue/niivue) - A WebGL module that can be embedded into HTML, Vue.js, Angular, React, Electron and Capacitor frameworks. It provides scripting and drag and drop abilities to visualize and render voxel-based images (e.g., DICOM and [NIfTI](https://nifti.nimh.nih.gov/)), meshes, and tractography streamlines and connectomes, as well as drawing functions ([demos](https://niivue.github.io/niivue/)).
```
Fixed the `NiiVue` entry to conform with the rest of the entries by adding a dash, and proposed a couple of minor changes.